src/main/java/io/cdap/e2e/utils/ElementHelper.java

```diff
@@ -0,0 +1,152 @@
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.e2e.utils;
+
+import org.apache.commons.lang3.StringUtils;
+import org.junit.Assert;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Element helper
+ */
+public class ElementHelper {
+  private static final Logger logger = LoggerFactory.getLogger(ElementHelper.class);
+  private static final JavascriptExecutor js = (JavascriptExecutor) SeleniumDriver.getDriver();
+  private static final Actions actions = new Actions(SeleniumDriver.getDriver());
+
+  public static void scrollToElement(WebElement element) {
+    logger.info("Scrolling to the element: " + element);
+    js.executeScript("arguments[0].scrollIntoView(true);", element);
+  }
+
+  public static void clickOnElement(WebElement element) {
+    scrollToElement(element);
+    WaitHelper.waitForElementToBeClickable(element);
+    logger.info("Click on the element: " + element);
+    element.click();
+  }
+
+  public static void clickOnElement(WebElement element, long timeoutInSeconds) {
+    scrollToElement(element);
+    WaitHelper.waitForElementToBeClickable(element, timeoutInSeconds);
+    logger.info("Click on the element: " + element);
+    element.click();
+  }
+
+  public static void clickOnElementUsingJsExecutor(WebElement element) {
+    logger.info("Click on the element using Javascript Executor: " + element);
+    js.executeScript("arguments[0].click();", element);
+  }
+
+  public static void sendKeys(WebElement element, String keys) {
+    scrollToElement(element);
+    WaitHelper.waitForElementToBeDisplayed(element);
+    logger.info("Send keys to element: " + element);
+    element.sendKeys(keys);
+  }
+
+  public static void hoverOverElement(WebElement element) {
+    scrollToElement(element);
+    WaitHelper.waitForElementToBeDisplayed(element);
+    logger.info("Hovering over the element: " + element);
+    actions.moveToElement(element).build().perform();
+  }
+
+  public static void clearElementValue(WebElement element, int limit) {
+    scrollToElement(element);
+    WaitHelper.waitForElementToBeDisplayed(element);
+    logger.info("Clearing WebElement: " + element);
+    boolean flag = false;
+    element.click();
+    element.clear();
+
+    for (int i = 0; i < limit; i++) {
+      if (element.getAttribute("value").equals("")) {
+        flag = true;
+        break;
+      }
+
+      element.sendKeys(Keys.BACK_SPACE);
+    }
+
+    if (!flag) {
+      Assert.fail("Element's current value is not cleared - " + element + " - "
+        + "Current value :" + element.getAttribute("value"));
+    }
```

Comment:
Since `replaceElementValue` is also using the same logic to clear the element value. Better to combine it within one common method and call it from both.

Reply:
Or may be call `clearElementValue` from `replaceElementValue` to first clear the element.

Reply:
Updated.